lib/html.js

```diff
@@ -254,19 +263,19 @@ var isMarkup = exports.isRaw = function(value) {
  * Determines if the value is an Array
  *
  * @private
- * @param {*} val the object being tested
- * @return {boolean}
+ * @param {any} val - parameter to be evaluated
+ * @return {boolean} true if an Array instance
  */
 var isArray = Array.isArray || function(val) {
   return (val instanceof Array);
 };
 
 /**
- * Determines if the value is a function
+ * Determines if the value is a Function
  *
  * @private
- * @param {*} val the object being tested
- * @return {boolean}
+ * @param {any} val - parameter to be evaluated
```

Comment:
making this consistent within the current file. in a future round of cleanup, this may be made more uniform throughout the repo